src/renderer/const.ts

```diff
@@ -69,3 +69,155 @@ export const AssetTKN18ERC20: Asset = {
 // This hardcode list is for testnet only
 export const ERC20Assets = [AssetUSDTERC20, AssetRuneEthERC20, AssetTKN8ERC20, AssetTKN18ERC20]
 export const ETHAssets = [AssetETH, ...ERC20Assets]
+
+export const routerABI = [
```

Comment:
Please move it into `renderer/services/const` and add a comment to link to contract https://ropsten.etherscan.io/address/0x9d496De78837f5a2bA64Cb40E62c19FBcB67f55a#code